liubov.tarasenko/homework3/prime.rb

```diff
@@ -0,0 +1,25 @@
+class PrimeMatrix
+  def initialize(max)
+    @max = max
+  end
+  
+  def prime_arr
+    base = [2,3,5,7].select { |x| x <= @max }    
+    rest = (2..@max).to_a.reject { |x| base.any? { |b| x % b == 0 } }
+    base + rest
+  end
+
+  def print_arr
+    puts calculate
+  end
+
+  def calculate
+    all = prime_arr
+    prime_matrix = Array.new(all.size) { |i| all.rotate(i) }
+    prime_matrix.map { |el| el.join(" ") }
```

Comment:
Let's have display logic separated from the calculation logic. 
Please move all display transformation to the print_arr method.
`calculate` should return data in the proper correct format, without any serialization

Reply:
thank you for review. just fixed it)